test/sanity/pep8/legacy-files.txt

```diff
@@ -1,3 +1,8 @@
+hacking/conf2yaml.py
+lib/ansible/constants.py
+lib/ansible/cli/config.py
+lib/ansible/config/data.py
+lib/ansible/config/manager.py
```

Comment:
I agree with sivel, these files need to be fixed rather than adding them to the testing blacklist.